src/components/Button/Button.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import './button.scss';
+
+export const Button = ({ props, text, styles }) => (
```

Comment:
Називати частину пропсів пропсами - не дуже гарна ідея
```suggestion
export const Button = ({ onClick, text, styles }) => (
```